CoEpi/storage/CENReport.swift

```diff
@@ -0,0 +1,27 @@
+import Foundation
+import RealmSwift
+
+struct CENReport : Codable {
+    var CENReportID: String = ""
+    var report: String = ""
+    var reportMimeType: String = ""
+    var reportTimestamp: Int = 0
+    var CENKeys: String = ""
+    var isUser: Bool = false
+    
+    func insert() -> Bool {
```

Comment:
We should have a `CENReportRepo` I think, same comment as above for `CEN` and `CENKey`